### PR TITLE
Feature/muscles

### PR DESCRIPTION
**Pull request by esrefbozkurt**
-new muscle groups component
-new home page 
-link to exercises page

### CODE REVIEW

File: components/Main/Exercises/StyledExercises.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ export const StyledExercises = styled.ul`
   flex-direction: column;
   border-bottom: inset #354f52 1px;
   margin: 10px;
-  border: 1px solid;
-  border-radius: 5px;
+  /* border: 1px solid; */
+  /* border-radius: 5px; */
```
Comment (julialoeschel) [inline components/Main/Exercises/StyledExercises.js:9]: please don`t push  commented out code to main :)

File: components/Main/Exercises/StyledExercises.js [hunk 2]
```diff
@@ -24,12 +24,13 @@ export const StyledExercises = styled.ul`
   h2 {
     font-family: Lato-bold;
     display: flex;
-
+    /* border-bottom: inset #354f52 1px; */
```
Comment (julialoeschel) [inline components/Main/Exercises/StyledExercises.js:27]: 👀

File: components/Main/MuscleGroups/MuscleGroupItem/MuscleGroupItem.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const MuscleGroupItem = ({ muscle }) => {
+  return (
+    <StyledMuscleGroupItem>
+      <Link href={`/exercises/${muscle}`}>
+        <li>
+          <Image
+            src={`/images/${muscle}.svg`}
+            width={158}
+            height={80}
+            alt="muscle"
```
Comment (julialoeschel) [inline components/Main/MuscleGroups/MuscleGroupItem/MuscleGroupItem.js:14]: I guess this is a special muscle, maybe use the full muscle name her ? `{ muscle }`

File: components/Main/MuscleGroups/StyledMuscleGroups.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  li {
+    cursor: pointer;
+    list-style: none;
+    width: 162px;
+    height: 112px;
+    color: rgba(53, 79, 82, 0.76);
+    /* font-size: 20px; */
```
Comment (julialoeschel) [inline components/Main/MuscleGroups/StyledMuscleGroups.js:13]: try to get rid of all the unused code

**Review by julialoeschel — COMMENTED** [review]
great job Eshi! 

I left some comments for you, 
small things. 

keep on going!!!
:octocat:

File: components/Main/MuscleGroups/MuscleGroupItem/MuscleGroupItem.js (hunk 3 above)
Comment (esrefbozkurt) [reply components/Main/MuscleGroups/MuscleGroupItem/MuscleGroupItem.js:14]: Thanks ! :)

File: components/Main/MuscleGroups/StyledMuscleGroups.js (hunk 4 above)
Comment (esrefbozkurt) [reply components/Main/MuscleGroups/StyledMuscleGroups.js:13]: Thank you, will do that :)

**Comment by esrefbozkurt** [comment]
> great job Eshi!
> 
> I left some comments for you, small things.
> 
> keep on going!!! :octocat:

Thank you so much!

**Pull request merged**